food-truck-api/src/main/resources/application.properties

```diff
@@ -11,5 +11,9 @@ spring.mail.password=${GOOGLE_SMTP_PASSWORD}
 spring.mail.properties.mail.smtp.auth=true
 spring.mail.properties.mail.smtp.starttls.enable=true
 
+hibernate.search.default.indexmanager=elasticsearch
+hibernate.search.default.elasticsearch.index_schema_management_strategy=drop-and-create
+hibernate.search.default.elasticsearch.required_index_status=yellow
+
```

Comment:
Is there something I'm missing and this is now actually dynamically connecting to either the dev or prod database that I'm not seeing?